production_configs/20220303_tuned_ze40_south/lstmcpipe_config.yml

```diff
@@ -0,0 +1,47 @@
+## MC PRODUCTION FULL PIPELINE CONFIGURATION ##
+#
+# Template example to request a MC prod.
+
+
+workflow_kind: lstchain  # Not expected to be changed by the user
+prod_id: tuned_ze40_south        # ex: local_no_n_islands. Default; v00 (if key left empty or None)
+
+base_path: /fefs/aswg/data/mc  # Path for centralised LST MC prods
+
+# Only for dl1 reprocessing - you will also need to change:
+# `r0_to_dl1` for `dl1ab` in "stages_to_be_run"
+#dl1_reference_id: # ex: 20220215_v0.9.1_prod5_trans_80_local_tailcut_8_4
+
+prod_type: prod5
+  # Choose between {prod5, prod3}
+obs_date: 20200629_prod5_trans_80
+  # prod5 : `20200629_prod5` or `20200629_prod5_trans_80`
+  # pro3 : `20190415`
+zenith: zenith_40deg  # Taken into account only when `prod_type: prod5` [ thus `obs_date` : 20200629_prod5(_trans_80) ]
+pointing: south_pointing
+particles:
+  - electron
+  - gamma
+  - gamma-diffuse
+  - proton
+offset_gammas:  # Taken into account only when `prod_type: prod5` [ thus `obs_date`: 20200629_prod5(_trans_80) ]
+  - off0.0deg
+  - off0.4deg
+
+source_environment:
+  source_file: /fefs/aswg/software/conda/etc/profile.d/conda.sh
+  conda_env: lstchain-v0.9.3
+
+stages_to_be_run:
+  - r0_to_dl1
```

Comment:
Change it to `dl1ab` for the DL1ab processing

Reply:
The thing is the DL1 for this version of lstchain doesn't exist at all, so should I add the '- dl1ab' line and keep '- r0_to_dl1', or just replacing will be enough?

Reply:
The DL1 file will be created from the R0 files, even when you write `dl1ab`. It is just to differentiate from the standard process of `r0_to_dl1` which has no DL1ab processing.

Reply:
ok, thanks!

Reply:
Hi all,
the `dl1ab` stage indeed runs the lstchain_dl1ab script for a (previous computed) set of dl1s.

The point is that it doesn't exists (yet- it is currently running) an updated MC prod for zen40deg. 

@LukasNickel will give an update of this stage on the next lst-reco meeting (and I will have a look later to all the configs before launching them). Thanks a lot for all the other reviewed points @chaimain :)

Reply:
Hi @garciagenrique , ok, so the r0->dl1 stage for 40deg zenith is running already (with default NSB I guess)? That's good! I thought it's not as I was asked to put the configuration files here first. Just ping me when it's done.